### DISCUSSION TITLE
An interesting textbook to link to

### DISCUSSION THREAD

**Discussion by rdstern** [IDEMSInternational/R-Instat#9762] [Climatic]
Here is a page:
![image](https://user-images.githubusercontent.com/11226469/82752417-ccde8d00-9db5-11ea-869b-f453c4ebd0a6.png)

You notice this generates some income from adverts.  They are different on each page, but each advert seems interesting and relevant.

However, that is not the aim here.  But this is a free (introductory) textbook and looks quite interesting.  It also provides a list of further accessible textbooks.

Perhaps we could link to these resources in the R-Instat help, or in the help file itself.

**Comment by Patowhiz** [comment]
I like this idea. It tallys in well with promotion of R-Instat in learning about statistics. Most of the undergraduate students will find this useful especially in their first year.

**Comment by lilyclements** [comment]
@rdstern @Patowhiz has this been done?

**Comment by rdstern** [comment]
@rachelkg this is an interesting topic on which I think we have done nothing since 2020.  I had a quick look at the website for this book, and note he has updated the text - maybe only slightly - in 2025.  We link to lots of things in the R-Instat help and our Moodle site, but not yet on this?

I wonder what we should do?  Can we discuss please.  Anyone else with suggestions?

**Comment by lilyclements** [comment]
An additional idea is that our loading pages are a bit outdated at the moment - could we add to these loading boxes a few new bits too, such as this interesting textbook as a resource, and mention the R-Instat moodle page and help options?

**Comment by AmsaleEjigu** [comment]
I stumbled upon this discussion page. @rachelkg is this book been added as suggested above?
